README.md

```diff
@@ -1,2 +1,54 @@
 # transifex-native-sandbox
 A collection of sample applications integrated with transifex-native
+
+# What is Transifex Native?
+
+A new way to localise software within the code - no files no deployments. A [video](https://www.youtube.com/watch?v=SzEOCKNjRZU) is worth a thousand words!
+But if you are a words person here is the [documentation](https://docs.transifex.com/transifex-native-sdk-overview/introduction).
+
+# Try it out yourself
+
+  
+  - [x] Step 1: Clone the repo of your preference (Python, django, JS, React or React Native)
+  
+
+  - [x] Step 2: Set things up using this guide: it's literally 3 commands. 
+  <ADD GUIDE here>
```

Comment:
This is going to be slightly different per sample app. I believe you can say something along the lines "follow the instructions within each README"

Reply:
can we keep the "it's literally 3 commands" or sth like that?